doc/changes.rst

```diff
@@ -26,18 +26,19 @@ desitarget Change Log
     * Useful to ensure ongoing compatibility with the mocks.
 * Bitmasks (1,12,13), rfiberflux cut for BGS Main Survey [`PR #562`_].
 * Implement a variety of fixes to `select_mock_targets` [`PR #561`_].
-* Fixes and updates to ``secondary.py`` [`PR #530`]:
+* Fixes and updates to ``secondary.py`` [`PR #530`_] and [`PR #574_`]:
     * Fix a bug that led to incorrect ``OBSCONDITIONS`` for secondary-only
```

Comment:
This way of updating the `changes` file is quick, but it also leads to PRs being out-of-order with versioning (i.e. `PR #574` should be included beneath version `0.35.0`, or a `no changes yet` version).

Please rewrite this so that `PR #574` has its own comment beneath the correct version (or `no changes yet` version) tag (which might just say, `fixes bugs in PR #530`).

Reply:
OK, thanks for the correction. 

I'm not up to speed on the protocol about whether to put the new PR under `0.35.0` vs. `no changes yet`. I merged with  the latest changes master and put this PR under `no changes yet`, hope that's ok.

Reply:
Yes, this should be under `no changes yet` (or, actually, should _replace_ the phrase `no changes yet`, as the first change for the next tag).

Can you fix a few malformed links, please? You have links like
```
[`PR #530_`]  [`PR #574_`]
```
these should be 
```
[`PR #530`_]  [`PR #574`_]
```

Reply:
Thanks, looks good now.